src/layout/CollapsibleList.js

```diff
@@ -0,0 +1,126 @@
+        this.pushListItem(new CollapsibleList().title("Tier 3: 1").testData3());
+        this.pushListItem(new CollapsibleList().title("Tier 3: 2").testData3());
+        return this;
+    };
+    CollapsibleList.prototype.testData3 = function () {
+        
+        this.pushListItem(
+            new Form()
+                .showSubmit(false)
+                .inputs([
+                    new Input().name("inp1").label("Input 1").type("textbox"),
+                    new Input().name("inp2").label("Input 2").type("textbox"),
+                    new Input().name("inp3").label("Input 3").type("textbox"),
+                ])
+        );
+        return this;
+    };
+    
+    CollapsibleList.prototype.pushListItem = function (widget) {
+        var contentArr = this.content();
+        contentArr.push(widget);
+        this.content(contentArr);
+        return this;
+    };
+
+    CollapsibleList.prototype.collapseClick = function (element) {
+        if(element.classed("closed")){
+            this._isClosed = false;
+            element.classed("open",true);
+            element.classed("closed",false);
+        } else {
+            this._isClosed = true;
+            element.classed("open",false);
+            element.classed("closed",true);
+        }
+    };
+
+    CollapsibleList.prototype.enter = function (domNode, element) {
+        HTMLWidget.prototype.enter.apply(this, arguments);
+        var context = this;
+        this._isClosed = this.collapsedByDefault();
+        element.classed(this._isClosed ? "closed" : "open",true);
+        
+        this.titleSpan = element.append("span").classed("collapsible-title",true);
+        this.iconDiv = element.append("div").classed("collapsible-icon",true);
+        this.ul = element.append("ul");
+        
+        this.icon = new FAChar().target(this.iconDiv.node());
+        
+        this.iconDiv.on("click",function(){
+            context.collapseClick(element);
+            context.render();
+        });
+        this.titleSpan.on("click",function(){
+            context.collapseClick(element);
+            context.render();
+        });
+    };
+
+    CollapsibleList.prototype.update = function (domNode, element) {
+        HTMLWidget.prototype.update.apply(this, arguments);
+        var context = this;
+        this.titleSpan.text(context.title().length > 0 ? context.title() : "CollapsibleList ["+context._id+"]");
+        var rows = this.ul.selectAll("li").data(this.content(),function(d) { return d._id; });
+        rows.enter()
+            .append(function(widget) {
+                var li = document.createElement("li");
+                if(widget._target === null){
+                    widget.target(li);
+                } else {
+                    return widget._target;
+                }
+                return li;
+            })
+        ;
+        rows.exit().remove();
+        rows.each(function(widget){
+            widget.render();
```

Comment:
I just tested by removing that code and its working fine?  Had you rebased on latest when you tested (as it needed https://github.com/hpcc-systems/Visualization/pull/843 to work)